package-system/python/darwin_x64/.p4ignore

```diff
@@ -0,0 +1,2 @@
+temp/
```

Comment:
Do we really need to add the .p4ignore files since this is being submitted to git?

Reply:
I'll delete this file.